wwwroot/faq.html

```diff
@@ -1,427 +1,400 @@
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "Why is it important to use a new address for every payment?",
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "<p>Addresses being used more than once is very damaging to privacy because that links together more blockchain transactions with proof that they were created by the same entity. The most private and secure way to use bitcoin is to send a brand new address to each person who pays you. After an address has received a coin, it should never be used again. Also, a brand new bitcoin address should be demanded from the recipient when sending bitcoin. Wasabi has a user interface which discourages address reuse by removing from the GUI addresses which have received a coin.</p><p>It has been argued that the phrase 'bitcoin address' was a bad name for this object because it implies it can be reused like an email address. A better name would be something like 'bitcoin invoice'.</p><p>Bitcoin isn't anonymous but pseudonymous, and the pseudonyms are bitcoin addresses. Avoiding address reuse is like throwing away a pseudonym after it has been used.</p>"
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "What is the passphrase used for?",
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "<p>The passhprase you set is used as a 13th seed word (as described in <a href='https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki' target='_blank'>BIP 39</a>) and to encrypt the private key of the extended private key (as described in <a href='https://github.com/bitcoin/bips/blob/master/bip-0038.mediawiki' target='_blank'>BIP 38</a>) to get an encrypted secret which is stored on the computer.</p><p>Wasabi Wallet stores only the BIP38 encrypted blob, so you'll need to type in the passphrase to spend or coinjoin from the wallet.</p><p>The passphrase will unlock your bitcoin to anyone who has access to the recovery words backup or the computer! If your backup gets compromised, this passphrase is the only thing protecting your precious sats.</p><p>It is important to use a random and long passphrase.</p>"
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "What are the fees for coinjoins?",
```

Comment:
IMO this should be moved up to be the 2nd FAQ.
If it doesn't matter just ignore my comment.